docs/index.md

```diff
@@ -0,0 +1,4 @@
+- [Runtime and interaction model for Plutus](runtime_and_interaction_model_for_plutus.md)
```

Comment:
Rename this file from `docs/index.md` to `docs/README.md` so that Github automatically populates it as the default when navigating to `docs`